features/product/managing_product_reviews/recalculating_product_average_rating.feature

```diff
@@ -11,15 +11,15 @@ Feature: Recalculating product average rating
         And this product has a review titled "Not bad" and rated 3 with a comment "Not bad car" added by customer "specter@teamharvey.com"
         And I am logged in as an administrator
 
-    @ui @api @javascript
```

Comment:
Why did you turn off these tests?

Reply:
because following by suggestion @GSadee (https://github.com/Sylius/Sylius/pull/11173#discussion_r388825373) i deleted from `ManagingProductReviewContext` step `And average rating of product "Lamborghini Gallardo Model" should be 4` (because it will be covered in product context). So, for now, this step isn't full implemented